packages/auth/tsconfig.json

```diff
@@ -12,7 +12,7 @@
     ],
     "moduleResolution": "node",
     "module": "es2015",
-    "target": "es5",
+    "target": "es6",
```

Comment:
Should this change be applied to other packages?

Reply:
I don't remember exactly what the problem was, but I think a dependency was using ES6. I'd do it on a per-package basis, but I don't really mind.

When we smoke test this across several browsers I'm sure we'll find issues